docs/content/dataset_standards.md

```diff
@@ -299,3 +299,26 @@ Statistical metrics are also computed as metadata for the individual datasets in
     | `mean` | `np.float64` | Mean value of the episode rewards.         |
     | `std`  | `np.float64` | Standard deviation of the episode rewards. |
     | `sum`  | `np.float64` | Total undiscounted return of the episode.  |
+
+## Observation and Action Spaces
+The Minari storage supports the following observation and action spaces:
+
+### Observation Spaces
+
+| Observation Space                                             | Description                                                  |
+| ------------------------------------------------------------- | ------------------------------------------------------------ |
+| [Discrete](https://github.com/Farama-Foundation/Gymnasium/blob/main/gymnasium/spaces/discrete.py) |  describes a discrete space where {0, 1, ..., n-1} are the possible values our observation or action can take. Values can be shifted to {a, a+1, ..., a+n-1} using an optional argument.|
+| [Box](https://github.com/Farama-Foundation/Gymnasium/blob/main/gymnasium/spaces/box.py)       | describes an n-dimensional continuous space. It's a bounded space where we can define the upper and lower limits which describe the valid values our observations can take |
```

Comment:
minor nitpick but this is slightly wordy, and might be better to just say something like "an n-dimensional continuous space. The `upper` and `lower` arguments can be used to define bounded spaces."

Reply:
Also missed a period at the end of this line

Reply:
ohh i just took the description from the gymnasium repo but fixing now

Reply:
Oh maybe that should be updated to be more clear too, probably not a huge priority but just figured since I saw it here might as well try to make it more understandable.